src/status_im2/contexts/quo_preview/profile/profile_card/profile_card.cljs

```diff
@@ -0,0 +1,57 @@
+(ns status-im2.contexts.quo-preview.profile.profile-card.profile-card
+  (:require [quo2.components.buttons.button :as button]
```

Comment:
this component should be in `quo2.components.profile.profile-card.view`, can you please move 🙏 .

You can see from the design file that is a part of the design components

Reply:
But in the other comment someone told me to extract it here. This was previously on the path you are saying

Reply:
hmm can you quote-reply or tag me in this comment. All I can see is a comment from @Parveshdhull  requesting the same change as me

Reply:
After conversation from discord, i did the change!